### PR TITLE
feat: Remove Learner Home from web crawling

### PR DESCRIPTION
**Pull request by leangseu-edx**
Ticket: https://2u-internal.atlassian.net/browse/AU-1056

This is probably the only way to do this. Our mfe is deployed on S3 and I am assuming dist is in the root of the bucket.

### CODE REVIEW

File: src/i18n/index.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import arMessages from './messages/ar.json';
 // no need to import en messages-- they are in the defaultMessage field
 import es419Messages from './messages/es_419.json';
 import frMessages from './messages/fr.json';
-import frcaMessages from './messages/fr_ca.json';
+import frcaMessages from './messages/fr_CA.json';
```
Comment (leangseu-edx) [inline src/i18n/index.jsx:5]: unrelated to this pr but this is necessary.

**Comment by mattcarter** [comment]
Approach seems reasonable. We should also include the stage environment too.

**Review by nsprenkle — APPROVED** [review]
Looks good, just curious where you got it from? Is this a pattern we use elsewhere?

**Comment by leangseu-edx** [comment]
> Looks good, just curious where you got it from? Is this a pattern we use elsewhere?

No. However, I think I should create a confluence page specifically for it.

**Comment by nsprenkle** [comment]
> > Looks good, just curious where you got it from? Is this a pattern we use elsewhere?
> 
> No. However, I think I should create a confluence page specifically for it.

Yeah, my dream is to have an MFE cookbook that explains these common build patterns.

**Pull request merged**